assets/js/hooks/cell_editor.js

```diff
@@ -38,6 +42,20 @@ const CellEditor = {
           const skeletonEl =
             editorContainer.querySelector(`[data-el-skeleton]`);
           skeletonEl && skeletonEl.remove();
+
+          switch (settings.editor_mode) {
+            case "emacs":
+              this.emacsMode = new EmacsExtension(this.liveEditor.editor);
+              this.emacsMode.start();
+              unregisterKey("Tab");
+              break;
```

Comment:
Probably just me, but I have a strong preference for if/else in JS, because the `break` makes the construct verbose :)